doc/userguide/devguide/contributing/backports-guide.rst

```diff
@@ -0,0 +1,121 @@
+========================
+Suricata Backports Guide
+========================
+
+This document describes the processes used to backport content to current stable
+Suricata releases. Most often, this means security and/or bug fixes;
+however, in some cases, features may be backported to previous Suricata releases.
+
+There are multiple versions of Suricata at any given time:
+    * Master
+    * Major stable release
+    * Old stable release
+
+For example, at the moment, there are 3 releases based on these Suricata branches:
+    * master: 8.0.0-dev, current development branch
+    * main-7.0.x: major stable release (note we're changing our naming conventions)
+    * master-6.0.x: old stable release
+
+For Suricata's release cadence and *end of life* policies, please check
+https://suricata.io/our-story/eol-policy/.
+
+The next sections discuss when and what to backport, and some guidelines when
+doing so.
+
+What should be backported?
+--------------------------
+
+Usually, when the team creates a ticket, we'll add the *Needs backport* related
+labels, so necessary backporting tickets will be automatically created. If you
+are working on a ticket that doesn't have such labels, nor backporting tasks
+associated, it probably doesn't need backporting. If you understand that the
+issue should be backported, please let us know in the ticket or related PR. But
+sometimes we'll miss those.
+
+The general principle used to determine what will be backported is:
+    * evasion/security fixes
+    * bug fixes
+    * in some cases, new features are backported if there are sufficient reasons to
+      backport a new feature.
+
+.. Note:: Exceptions
+
+    There can be cases where backports may be "missed" -- some issues may not be
+    labeled as needing backports and some PRs may be merged without an issue.
+
+    This guide may be insufficient for some situations. When in doubt, please reach
+    out to the team on the backport ticket or PR.
+
+Selection overview
+------------------
+
+All items considered for backports should be reviewed with the following:
+    * risk estimate: will the change introduce new bugs? Consider the scope and
+      items affected by the change.
+    * behavioral change: how much will the behavior of the system be changed by the
+      backport. for example, a small change to decode additional encapsulation
+      protocols may result in more traffic being presented to Suricata.
+    * default settings: if the issue alters behavior, can it be made optional, and
+      at what cost?
+
+Creating backport tickets -- new issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Redmine: for security/evasion and bug fixes, when creating a new redmine issue,
```

Comment:
we have a security ticket type, so lets just refer to that and not add evasion here (and in other places)